adops/step-by-step.md

```diff
@@ -1,9 +1,241 @@
+
+{: .alert.alert-info :}
+For deals, the Rate portion of this value will contain the dealID if deals are prioritized. See [Getting Started with Long Form Video](/prebid-video/video-long-form.html#configuration) for engineering information.
+
+{: .alert.alert-info :}
+Engineers will need to include the [Adpod module](/dev-docs/modules/adpod.html) and the [Category Translation module](/dev-docs/modules/categoryTranslation.html) in Prebid.js to implement long-form video bidding.
+
+### Creative-level Targeting
+
+In the **Expected Creatives** section, you can add targeting that applies to creatives rather than to the entire line item. For Prebid you might want to do this if you’re going to use a single line item for multiple formats, or if you have multiple video cache locations.
+
+To set creative-level targeting, do the following:
+
+1. In the line item's **Expected creatives** box, enter the creative size or sizes.
+2. Click **Show creative details** (for display) or **Expand all** (for video).
+3. In the first creative size box, under **Creative targeting** click **Add targeting** and select **Add new targeting**. This slides out the **Creative targeting** window.
+4. Expand **Custom targeting** and enter the appropriate key values.
+
+Repeat the preceding steps for each creative in the line item.
+
+### Save the Line Item
+
+You’ve now added all fields necessary for targeting Prebid line items. You can add any other line item options you would normally use, such as additional targeting for geography. When you’ve filled in all the above fields, click **Save** to save your line item.
+
+## Create Creatives
+
+The process you use to create your creatives differs based on the media type. Follow the instructions for the appropriate media type:
+
+- [Banner/Outstream/AMP](/adops/gam-creative-banner-sbs.html)
+- [Native](/adops/gam-native.html)
+- [Video](/adops/setting-up-prebid-video-in-dfp.html)
+
+
+## Duplicate Creative
+
+After you've created your creatives, you’ll need to associate a creative with each size in your line item. Even if you’ve specified only one or two sizes, you might actually want more creatives than you have sizes. Because the creative body itself is identical no matter which size you’re associating it with, you can duplicate the creative so you have as many as you need.
+
+{: .alert.alert-info :}
+You need extra copies of the creative because GAM will display only one creative per line item per page. So if you have two ad slots where two Prebid bids in the same price bucket win the auctions, if you defined only one creative on that line item, only that one creative would be displayed in an ad slot, the other ad slot would fall through to your house ads. See [Creative Considerations](/adops/creative-considerations.html) for more information.
```

Comment:
This is really hard to follow, too long of a sentence, too many commas?
Do we even need the further explanation of 
"So if you have two ad slots where two Prebid bids in the same price bucket win the auctions, if you defined only one creative on that line item, only that one creative would be displayed in an ad slot, the other ad slot would fall through to your house ads."